jenkins/release_ci/opensearch_maven_release/maven-sign-release.jenkinsfile

```diff
@@ -0,0 +1,94 @@
+lib = library(identifier: 'jenkins@20211123', retriever: legacySCM(scm))
+
+pipeline {
+    agent {
+        docker {
+            label 'Jenkins-Agent-al2-x64-c54xlarge-Docker-Host'
+            image 'opensearchstaging/ci-runner:centos7-x64-arm64-jdkmulti-node10.24.1-cypress6.9.1-20211130'
+            alwaysPull true
+        }
+    }
+    parameters {
+        string(
+            defaultValue: 'x64',
+            name: 'ARCH',
+            trim: true
+        )
+        string(
+            defaultValue: 'linux',
+            name: 'PLATFORM',
+            trim: true
+        )
+        string(
+            name: 'BUILD_ID',
+            trim: true
+        )
+        string(
+            name: 'VERSION',
+            trim: true
+        )
+    }
+    environment {
+        ARTIFACT_PATH = "distribution-build-opensearch/${params.VERSION}/${params.BUILD_ID}/${params.PLATFORM}/${params.ARCH}/builds"
```

Comment:
don't use `params.ARCH`, only `ARCH` would also work. using `params.ARCH` will create problems when setting variable bindings in tests

Reply:
I think using `param.ARCH` is more reasonable than just `ARCH`. What issue would it cause for testing?

Reply:
`parms.ARCH` variables are not set and picked up properly when used with bindings. I faced a similar problem when writing tests for ecr.